src/view/translation.js

```diff
@@ -827,5 +827,73 @@ Candy.View.Translation = {
 		'errorMaxOccupantsReached': 'No pots unir-te a la sala "%s": hi ha masses participants.',
 
 		'antiSpamMessage' : 'Si us plau, no facis spam. Has estat bloquejat temporalment.'
+	},
+	'cs' : {
+		'status': 'Stav: %s',
+		'statusConnecting': 'Připojování...',
+		'statusConnected' : 'Připojeno',
+		'statusDisconnecting': 'Odpojování...',
+		'statusDisconnected' : 'Odpojeno',
+		'statusAuthfail': 'Přihlášení selhalo',
+
+		'roomSubject'  : 'Předmět:',
+		'messageSubmit': 'Odeslat',
+
+		'labelUsername': 'Už. jméno:',
+		'labelNickname': 'Přezdívka:',
+		'labelPassword': 'Heslo:',
+		'loginSubmit'  : 'Přihlásit se',
+		'loginInvalid'  : 'Neplatné JID',
+
+		'reason'				: 'Důvod:',
+		'subject'				: 'Předmět:',
+		'reasonWas'				: 'Důvod byl: %s.',
+		'kickActionLabel'		: 'Vykopnout',
```

Comment:
Sure, I copied English translation only :)
Shall I fix it in whole file ? Just replace this regexp: s/\s+:/:  so at least one space followed by : replace with just :